### PR TITLE
Include Android

### PR DESCRIPTION
**Pull request by kloggy**
I guess the icon needs to change too but I didn't know how I would include a new image.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase:
- This PR fixes or closes issue:

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

**Comment by klaasnicolaas** [comment]
In this case we should then also change the icon and url

File: source/docs/index.markdown [hunk 1]
```diff
@@ -40,7 +40,7 @@ The documentation covers beginner to advanced topics around the installation, se
     <div class='img-container'>
       <img src='/images/supported_brands/apple.png' />
```
Comment (klaasnicolaas) [inline source/docs/index.markdown]: I think we should change this image also, for something that presents a mobile 📱

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]

File: source/docs/index.markdown [hunk 2]
```diff
@@ -40,7 +40,7 @@ The documentation covers beginner to advanced topics around the installation, se
...
     </div>
-    <div class='title'>iOS</div>
+    <div class='title'>Android and iOS</div>
```
Comment (frenck) [inline source/docs/index.markdown:43]:
```suggestion
    <div class='title'>Mobile Apps</div>
```

**Comment by frenck** [comment]
The link of the item is also incorrect, it should go to `https://companion.home-assistant.io`

**Comment by kloggy** [comment]
I agree with the comments from both @klaasnicolaas and @frenck but as a GitHub novice I am not sure if this is waiting for me to do anything.

Sorry for my ignorance...

**Review by frenck — APPROVED** [review]
👍

**Pull request merged**